riscof/utils.py

```diff
@@ -598,8 +598,16 @@ def riscof_cmdline_args():
                         metavar= 'PATH',
                         default=str(pathlib.Path('./riscof_work').absolute())
                         )
-    run.add_argument('--testfile',action='store_true',
-                     help="Run tests from already generated testlist file")
+    run.add_argument('--dbfile',
+                        type= lambda p: str(pathlib.Path(p).absolute()),
+                        action='store',
+                        help='The Path to the custom suite directory.',
```

Comment:
The help message for `dbfile` and `testfile` arguments are wrong.

Reply:
Please update the documentation in `commands.rst` and `quickstart.rst` according to the recent changes too.

Reply:
sure on it

Reply:
Done